boshio/boshio.go

```diff
@@ -48,24 +48,24 @@ func (s Stemcell) Details() Metadata {
 
 type Client struct {
 	Host                 string
-	bar                  bar
-	ranger               ranger
-	stemcellMetadataPath string
-	stemcellDownloadPath string
+	Bar                  bar
+	Ranger               ranger
+	StemcellMetadataPath string
+	StemcellDownloadPath string
 }
 
 func NewClient(b bar, r ranger) *Client {
```

Comment:
My concern was with these `bar` and `ranger` types being private, not the fields they're used in.